examples/CustomCharacter/CustomCharacter.ino

```diff
@@ -3,68 +3,33 @@
 
 // создаем объект для работы с дисплеем
 TroykaTextLCD lcd;
+// по умолчанию в параметры конструктора дисплея зашиты параметры:
+// выбор шины I²C (&Wire), I²C-адрес и пин подсветки
+// TroykaTextLCD lcd(&Wire, 0x3E, 7);
 
 // создаём массивы иконок
-byte heart[8] = {
-  0b00000,
-  0b01010,
-  0b11111,
-  0b11111,
-  0b11111,
-  0b01110,
-  0b00100,
-  0b00000
-};
+byte heart[8] = {0b00000, 0b01010, 0b11111, 0b11111,
+                 0b11111, 0b01110, 0b00100, 0b00000};
```

Comment:
You can print your array of 8 lines because it`s a picture.

Reply:
Igor. Change please your MSCode settings - use .clang.format from repo and rearrange code.

Reply:
ok